### PR TITLE
Remove ssh address references

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

First step on the legacy runner removal. Remove `ssh_address` field references.

**Important: once this is merged the checks with the legacy runner won't work anymore (the "demo" env will continue working as it fakes this communication)**

## How was this tested?

Nothing to be done

### CODE REVIEW

File: lib/trento/application/integration/checks/adapter/runner.ex [hunk 1]
```diff
@@ -57,7 +57,7 @@ defmodule Trento.Integration.Checks.Runner do
         Enum.map(host_settings, fn host ->
           %{
             host_id: host.host_id,
-            address: host.ssh_address,
+            address: "",
```
Comment (arbulu89) [inline lib/trento/application/integration/checks/adapter/runner.ex:60]: Setting this empty by now. This complete function will be removed once we remove the connection settings code. I didn't want to do it here, to not overload the PR more

File: lib/trento_web/views/prometheus_view.ex [hunk 2]
```diff
@@ -10,7 +10,7 @@ defmodule TrentoWeb.PrometheusView do
 
   def render("target.json", %{target: target}) do
     %{
-      targets: ["#{target.ssh_address}:#{@node_exporter_port}"],
+      targets: ["#{List.first(target.ip_addresses, target.hostname)}:#{@node_exporter_port}"],
```
Comment (arbulu89) [inline lib/trento_web/views/prometheus_view.ex:13]: Use the 1st obtained IP address (or the hostname as fallbak) as node exporter address. I think it should be enough, but if in the future we need more customization, we will need to add back some settings to be added by the user maybe

File: lib/trento/application/integration/checks/adapter/runner.ex (hunk 1 above)
Comment (fabriziosestito) [reply lib/trento/application/integration/checks/adapter/runner.ex:60]: could you add a FIXME reminder?

File: lib/trento/domain/host/events/host_registered.ex [hunk 3]
```diff
@@ -9,7 +9,6 @@ defmodule Trento.Domain.Events.HostRegistered do
     field :host_id, Ecto.UUID
     field :hostname, :string
     field :ip_addresses, {:array, :string}
```
Comment (fabriziosestito) [inline lib/trento/domain/host/events/host_registered.ex:11]: We need to agree on when we will upcast or, in general, what we consider a breaking change. This is valid for API versioning but also event versioning (internal events and integration events).
My proposal is to go like this but track it somewhere.

File: lib/trento/application/integration/checks/adapter/runner.ex (hunk 1 above)
Comment (arbulu89) [reply lib/trento/application/integration/checks/adapter/runner.ex:60]: The complete file will be gone once we finish removing the legacy runner usage. 
I'm actually already working on the removal in other branches. I don't think we will forget anyhow about this

File: lib/trento/domain/host/events/host_registered.ex (hunk 3 above)
Comment (arbulu89) [reply lib/trento/domain/host/events/host_registered.ex:11]: @fabriziosestito I will create a ticket in our backlog to discuss this.
I think that we might agree that in the particular scenario (removing a field), we don't have any breaking change, so upgrading the version would only be "informative"

File: lib/trento_web/views/prometheus_view.ex (hunk 2 above)
Comment (fabriziosestito) [reply lib/trento_web/views/prometheus_view.ex:13]: Could we track this in a new card as well?

Comment (arbulu89) [reply lib/trento_web/views/prometheus_view.ex:13]: Done

**Review by fabriziosestito — APPROVED** [review]

**Pull request merged**